zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/LazyOpenInterpreter.java

```diff
@@ -112,6 +113,12 @@ public int getProgress(InterpreterContext context) {
   }
 
   @Override
+  public List<InterpreterProgressInfo> getProgressInfo(InterpreterContext context) {
+    open();
```

Comment:
To initialize interpreter, if not initialized, similar to `getProgress()` and `completion()`.